### ISSUE TITLE
Snapdragon: segfault

### ISSUE THREAD

**Issue by julianoes** [PX4/Firmware#4636]
Something broke poor Snappy:

```
pxh> Segmentation fault
```

**Comment by julianoes** [comment]
sdlog2 seems to be at fault:

```
pxh> sdlog2 start -r 100 -e -t -a -b 200
WARN  log buffer size: 204800 bytes
INFO  [blackbox] /root/log/2016-05-25
INFO  [blackbox] recording: 18_50_40.px4log
pxh> Segmentation fault
```

**Comment by julianoes** [comment]
Fixed by #4638.
